### PR TITLE
Refresh the Access Controls menu

### LINKED ISSUE #19933: Configure Access -> Manage Access
Screenshot 2023-01-05 at 3 19 07 PM

Nit: configure access is a subset of this section does. Standard term is manage access as in "Identity and access management".

### PR DESCRIPTION
**Pull request by ptgott**
Closes #19933

- Changes the "Configure Access" menu item to "Manage Access"
- The Access Controls menu page was out of date, so I changed it to reflect the current sequence of subsections. This more narrative approach also puts each subsection in the context of a user's overall Teleport setup.

### CODE REVIEW

**Review by zmb3 — APPROVED** [review]

File: docs/config.json [hunk 1]
```diff
@@ -205,7 +205,7 @@
     },
     {
       "icon": "lock",
-      "title": "Configure Access",
+      "title": "Manage Access",
```
Comment (alexfornuto) [inline docs/config.json:208]: Nit: Are we at all worried about the two similar titles seeming redundant? If not, ignore me.
image

**Review by alexfornuto — APPROVED** [review]
Neat! I made several suggestions, but nothing that blocks this from merging if needed, so 👍

File: docs/pages/access-controls/introduction.mdx [hunk 2]
```diff
@@ -1,40 +1,80 @@
...
-description: Provide Role-Based Access Control (RBAC) for SSH, Kubernetes, Databases, and Web Apps.
-h1: Access Controls for SSH, Kubernetes, Databases and Web Apps
+title: Manage Access to your Cluster
+description: How to provide role-based access control (RBAC) for servers, databases, Kubernetes clusters, and other resources in your infrastructure
+layout: tocless-doc
 ---
 
-## Introduction
+After [deploying a Teleport cluster](../deploy-a-cluster/introduction.mdx), the
+next step is to manage the access that Teleport users have to resources in your
```
[the remark this answers is not in the document]
Comment (ptgott) [reply docs/pages/access-controls/introduction.mdx:8]: I think "that" works a bit better here

File: docs/pages/access-controls/introduction.mdx [hunk 3]
```diff
@@ -1,40 +1,80 @@
...
+Configure Access Controls in a five-minute [Getting Started](./getting-started.mdx)
+guide.
 
-## Getting started
+## Set up Teleport roles
 
-Configure Access Controls in a 5 minute [Getting Started](./getting-started.mdx)
-guide.
+The heart of Teleport's RBAC system is the **role**, a configuration document
+that specifies access restrictions to resources in your Teleport cluster.
```
[the remark this answers is not in the document]
Comment (ptgott) [reply docs/pages/access-controls/introduction.mdx:31]: Using "policies" instead of "restrictions"

File: docs/config.json (hunk 1 above)
Comment (ptgott) [reply docs/config.json:208]: I really, really want to reorganize the "Manage your Cluster" section, so hopefully we won't have this redundancy for long!

**Pull request merged**